### DISCUSSION TITLE
How to remove my extra solutions of one Kata

### DISCUSSION THREAD

**Discussion by ladonin** [codewars/codewars.com#3172] [Q&A] [answered]
Steps:
- Made code, pressed Attempt, pressed Submit => my code booted up to Solutions.
- Found a flaw, corected, then pressed Attempt and Submit

Then now Solutions in the Kata have 2 of my "codes"... (it can be infinite solutions of mine).

How can I remove any of my solutions in the Kata?
I want to keep only last Solution.

**Comment by hobovsky** [comment, marked as answer]
It is not possible to delete your own solutions. They are going to stay, but it's not much of a problem from the perspective of the system. Possibility of deleting solutions was already [discussed](https://github.com/codewars/codewars.com/issues/540), more than once, but currently there is no plan to introduce such functionality.

**Reply by ladonin** [reply]
Thanks!

**Reply by AnatolyGavrilov** [reply]
> It is not possible to delete your own solutions. They are going to stay, but it's not much of a problem from the perspective of the system. Possibility of deleting solutions was already [discussed](https://github.com/codewars/codewars.com/issues/540), more than once, but currently there is no plan to introduce such functionality.


Is there a way to at least hide them?

**Comment by jessya020** [comment]
To manage your submitted solutions in a Kata (a common term for coding exercises on platforms like Codewars) and retain only your most recent version, follow this clear, step-by-step guide:
First, navigate to the specific Kata where you have multiple solutions. Start by accessing your user profile (usually labeled “My Profile” or your username in the site’s header). From there, look for sections like “My Solutions,” “Submitted Katas,” or “My Code” — these will let you filter and find the Kata you worked on (you can also search for the Kata by name if the platform has a search bar).
Once you’re on the Kata’s page, locate the list of your submitted solutions. Most platforms organize these by timestamp, with older solutions appearing above the latest one (e.g., labeled “Submitted 3 hours ago” or “Submitted 1 hour ago”). There may be a dedicated tab like “My Solutions” or a dropdown menu labeled “Your Submissions” to access this list.
Next, delete the old solutions you want to remove. Hover over each older solution entry — you’ll typically see a delete option, such as a trash can icon, a “Delete” button, or a three-dot menu (⋮) that expands to show “Remove Solution” or “Delete This Version.” Click this option, then confirm the deletion when prompted (many sites add a confirmation step, like “Are you sure you want to delete this solution?” to avoid accidental removal).
Finally, verify that only your latest solution remains. Refresh the Kata page and check the “My Solutions” section again — it should now display only your most recent submission. If you don’t see a delete button immediately, try opening the old solution first (by clicking its timestamp or “View Code”) — some platforms place the delete function within the solution’s detail page. If the delete option is still missing, check the platform’s help center (usually in the footer under “Help” or “FAQ”) and search for terms like “delete submitted solution” to find platform-specific tips.
